build.gradle

```diff
@@ -16,6 +16,10 @@ buildscript {
     }
 }
 
+plugins {
+  id("org.sonarqube") version "4.3.1.3277"
+}
+
```

Comment:
This should be in `lib/build.gradle`
The root `build.gradle` is only needed to declare sources for dependencies/plugins

Reply:
@mdrlzy good idea, could you move it there?